marconi-mamba/README.md

```diff
@@ -1,78 +1,174 @@
-marconi-mamba
---
 
-This is an initial draft for the Marconi Mamba README file. Going forward, we will need to discuss and refine this document and/or the implementation so that they match.
+# Marconi-Mamba
 
-## Description
-marconi-mamba is a [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) HTTP server on top of the [marconi](../marconi/README.md)
+Marconi-Mamba is a lightweight, customizable chain follower application and library for DApp developers who need to index and query the Cardano blockchain. Marconi-Mamba provides a way for DApps to have access to data that has already been stored on-chain by indexing portions of the blockchain.
 
-## Playing with the JSON-RPC
-There is an example client/server JSON-RPC to experiment with the JSON-RPC clien/server. Here are the steps to build and execute the examples:
+## Purpose
+
+The purpose of Marconi-Mamba is to make the core Marconi APIs available to non-Haskell applications.
+
+## Interface
+
+The interface for Marconi-Mamba uses [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) over HTTP built on top of [Marconi](../marconi/README.md).
 
-#### Build the project
 
-``` sh
-cabal build all
-cd marconi-mamba
-cabal build
 ```
+             Running on a single machine                    Internet
```

Comment:
Did you use a specific tool to build this ASCII diagram?

Reply:
@koslambrou I used emacs [artist mode](https://www.emacswiki.org/emacs/ArtistMode#:~:text=Artist%20is%20an%20Emacs%20lisp,%2C%20%2D%2C%20%2F%20and%20%5C.)

Reply:
We can work with a graphic designer from the Content Team when we need nice graphics. I'm thinking this current version may be good for the README file for now. But I'll ask the designer (Lee Rosney) if he could take a look and come up with a design for this.

Reply:
I actually prefer ASCII diagrams than any other modern graphics. I'm proposing we keep this in the medium term, since we may need to update it. Also, if we create other diagrams, we can stay consistent.